### ISSUE TITLE
Popover displacement bug

### ISSUE THREAD

**Issue by nikolaypronchev** [nathanreyes/v-calendar#908]
**v-calendar** `2.3.0`
**vue** 2.6.11
**browser** Google Chrome 91.0.4472.77 (x64)
**Description**
If fixed positioned datepicker is located inside container...
```css
position: absolute;
overflow: hidden;
left: 200px; //any shift
```
...a popover position bug occurs. When you click the month selection button, the popup that appears is shifted (No. 1 in the image). After hovering the cursor, the window returns to the correct position (No. 2 in the image).

![bug](https://user-images.githubusercontent.com/55616107/120510082-bfe57800-c3d1-11eb-9fd4-2aa7d5e42c29.png)

[https://codesandbox.io/s/laughing-shape-g5okd?file=/src/App.vue](url)
Hover input -> Click month name -> Popover is shifted right -> Hover popover -> It shifts to correct position

**UPD** Even after hovering, it is not correctly positioned

**Comment by Chernavskikh** [comment]
+ 1 for this bug

**Comment by nikolaypronchev** [comment]
> temporary solution:
> 
> ```
> .vc-popover-content-wrapper.is-interactive .vc-popover-content-wrapper {
>   transform: translate3d(43px, 38px, 0) !important;
> }
> ```
Yeah, this works fine. Seems like it's not Popper.js issue, as it correctly calculates popover position in same situation
